pkg/agent/clientset.go

```diff
@@ -38,42 +38,67 @@ const (
 
 // ClientSet consists of clients connected to each instance of an HA proxy server.
 type ClientSet struct {
-	mu      sync.Mutex         //protects the clients.
-	clients map[string]*Client // map between serverID and the client
-	// connects to this server.
-
-	agentID string // ID of this agent
-	address string // proxy server address. Assuming HA proxy server
-
-	leaseCounter            ServerCounter // counts number of proxy server leases
-	lastReceivedServerCount int           // last server count received from a proxy server
-	lastServerCount         int           // last server count value from either lease system or proxy server, former takes priority
-
-	// unless it is an HA server. Initialized when the ClientSet creates
-	// the first client. When syncForever is set, it will be the most recently seen.
-	syncInterval time.Duration // The interval by which the agent
-	// periodically checks that it has connections to all instances of the
-	// proxy server.
-	probeInterval time.Duration // The interval by which the agent
+	// mu guards access to the clients map
+	mu sync.Mutex
+
+	// clients is a map between serverID and the client
+	// connected to this server.
+	clients map[string]*Client
+
+	// agentID is "our ID" - the ID of this agent.
+	agentID string
+
+	// Address is the proxy server address.  Assuming HA proxy server
```

Comment:
TODO to improve this. Its the address we use to contact the ANP Server. Normally this would be a LB address in front of the ANP Servers.